### DISCUSSION TITLE
External MQTT target

### DISCUSSION THREAD

**Discussion by NoroNetwork** [wmbusmeters/wmbusmeters#1120] [Q&A]
Hello, 

My setup:
Fedora server install. 
Instaled wmbusmeters 
and  config: 
loglevel=debug
device=auto:c1,t1
logtelegrams=true
format=json
shell=/usr/bin/mosquitto_pub -h 192.168.200.13 -t wmbusmeters/"$METER_ID" -m "$METER_JSON"
logfile=/var/log/wmbusmeters/wmbusmeters.log

But its not working... 

I get this error: 

(shell) exec "/bin/sh"
(shell) arg "-c"
(shell) arg "/usr/bin/mosquitto_pub -h 192.168.200.13 -t wmbusmeters/"$METER_ID" -m "$METER_JSON""
(shell) env "METER_JSON={"media":"water","meter":"apator162","name":"TeplaVoda","id":"03060585","total_m3":58.654,"timestamp":"2023-12-08T13:15:08Z","device":"rtlwmbus[00000001]","rssi_dbm":143}"
(shell) env "METER_ID=03060585"
(shell) env "METER_NAME=TeplaVoda"
(shell) env "METER_MEDIA=water"
(shell) env "METER_TYPE=apator162"
(shell) env "METER_TIMESTAMP=2023-12-08T13:15:08Z"
(shell) env "METER_TIMESTAMP_UTC=2023-12-08T13:15:08Z"
(shell) env "METER_TIMESTAMP_UT=1702041308"
(shell) env "METER_TIMESTAMP_LT=2023-12-08 14:15.08"
(shell) env "METER_DEVICE=rtlwmbus[00000001]"
(shell) env "METER_RSSI_DBM=143"
(shell) env "METER_TOTAL_M3=58.654"
(shell) waiting for child 2331 to complete.
(shell) /bin/sh: return code 127
[2023-12-08_14:15:08] (shell) /bin/sh exited with non-zero return code: 127

So as i understand i get all the thing i need to transfet over MQTT but i am missing mqtt??

I have Home assistant setup on diferent machine: 192.168.200.13 

So how i send the data to my Mosquito broker? 

Thanks :)

**Comment by BIBOLV** [comment]
Hi,
Try posting something manually using /usr/bin/mosquitto_pub, that way you should be able to identify errors if there are any.
